components/board/hat/pca9685/pca9685.go

```diff
@@ -212,6 +212,16 @@ func (pca *PCA9685) frequency(ctx context.Context) (float64, error) {
 	return float64(pca.referenceClockSpeed) / 4096.0 / float64(prescale), nil
 }
 
+// StreamTicks starts a stream of digital interrupt ticks.
+func (pca *PCA9685) StreamTicks(ctx context.Context, interrupts []string, ch chan board.Tick, extra map[string]interface{}) error {
+	return grpc.UnimplementedError
+}
+
+// RemoveCallbacks removes the callbacks from the given interrupts.
+func (pca *PCA9685) RemoveCallbacks(ctx context.Context, interrupts []string, ch chan board.Tick) error {
+	return grpc.UnimplementedError
+}
+
```

Comment:
or on any other board

Reply:
my bad forgot to delete these, don't need this function in every board driver anymore.